rc/extra/autorestore.kak

```diff
@@ -9,32 +9,36 @@ def autorestore-restore-buffer -docstring "Restore the backup for the current fi
 
         ## Find the name of the latest backup created for the buffer that was open
         ## NB. kak doesn't automatically cd to current buffer_dirname.
-        buffer_backup_glob="${buffer_dirname}/.${buffer_basename}.kak.*"
-        backup_path=$(ls -1t ${buffer_backup_glob} 2>/dev/null | head -n 1)
+        buffer_backup_glob=".${buffer_basename}.kak.*"
+#        backup_path=$(ls -1t ${buffer_backup_glob} 2>/dev/null | head -n 1)
+        newer=$(find "${buffer_dirname}" -maxdepth 1 -name "${buffer_backup_glob}" -newer "${kak_buffile}" 2>/dev/null | head -n1)
+        older=$(find "${buffer_dirname}" -maxdepth 1 -name "${buffer_backup_glob}" \! -newer "${kak_buffile}" 2>/dev/null | head -n1)
 
-        if [ -z "${backup_path}" ]; then
-            exit
-        elif [ ${kak_buffile} -nt ${backup_path} ]; then
-          printf "%s\\n" "echo -debug \"Old backup file found: ${backup_path}; will not restore.\""
-          exit
+
+        if [ -z "${newer}" ];
+        then
+          if [ -n "${older}" ];
+          then
```

Comment:
You could save a few lines here by writing `if [ -z "${newer}" ] && [ -z "${older}" ]; then`

Reply:
Cases:
1. no backup file.
2. no newer backup file; no older backup file.
3. no newer backup file; 1 or more older backup files.